# 2.6 Dimensionlal Homogeneity and Dimensionless Quantities

If quantities can only be added and subtracted when units match, same is true of dimensionless quantities

*Every valid equation msut be dimensionally homogeneous: that is, all additive terms on both sides of the equation must have the same dimensions.*

Consider the equation:
$$
u(\text{m/s}) = 
u_0(\text{m/s}) + g(\text{m} / \text{s}^2) \times t(\text{s})
$$

It is dimensionally homogeneous since each term is length/time. It also has consisten units since each term is m/s. If equation is dimensionally homogeneous but units are inconsistent, the terms must be converted to match units. For example, if time, t, was given in minutes:

$$
u(\text{m/s}) = 
u_0(\text{m/s}) + 
g(\text{m} / \text{s}^2) \times 
t(\text{min}) \times 
(60 \text{ s/min})
$$

$$ = u_0 + 60 g t$$

## Example 2.6-1 Dimensional Homogeneity

Consider the equation:
$$ D(\text{ft}) = 3 t(\text{s}) - 4 $$

### Problem 1
If the equation is valid, what are the dimensions of the constants 3 and 4?
### Solution 1
4 has dimensions of *length* and 3 has dimensions *length/time*

### Problem 2
If the equation is consistent in units, what are the units of the constants 3 and 4?
### Solution 2
4 has units feet (ft) and 3 has units feet per second (ft/s)

### Problem 3
Derive an equation for distance in meters in terms of time in minutes
### Solution 3

In [3]:
import pint
ureg = pint.UnitRegistry()

print((1 * ureg.meter).to(ureg.foot))
print((1 * ureg.minute).to(ureg.second))

3.2808398950131235 foot
60.0 second


$$D = D' \times 3.28$$
$$t = t' \times 60$$

Substitute into equation
$$
3.28 \times D' = 
3 \times 60 \times t' + 4
$$
or
$$ D' = 55 \times t' + 1.22$$

To rewrite an equation in terms of new variables:
- Define new variables (add 'primes' to old variable names)
- Write expressions for each old variable of the corresponding new variable
- Substitute these expressions in the original equation and simplify

A **dimensionless quantity** can be a pure number (2, 1, 3, 5/2) or multiplicative combination with no net dimensions:

$$ 
\frac{M(\text{g})}{M_o(\text{g})} 
\frac{D(\text{cm})u(\text{cm/s})\rho(\text{g/cm}^3)}
     {\mu[\text{g/(cm-s)}]}
$$

A quantity such as $M/M_o$ or $Du\rho/\mu$ is also called a **dimensionless group**.

Exponents, transcendental functions, and arguments of transcendental functions must be dimensionless quantities. $10^{2\text{ft}}$ and $\text{sin }(8 \text{ s})$ doesn't make sense.

### Example 2.6-2 Dimensional Homogeneity and Dimensionless Groups

A quantity, $k$, depends on the temperature $T$ in the following manner:
$$
k(\frac{\text{mol}}{\text{cm}^3\text{s}}) =
1.2 \times 10^5 \text{exp}(-\frac{20,000}{1.987T})
$$

Units of the quantity 20,000 are cal/mol, and T is in K (kelvin). What are the units of $1.2 \times 10^5$ and $1.987$?

- Since argument of exp must be dimneionless, 1.987 must be of units $\text{cal/(mol K)}$
- Units of $1.2 \times 10^5$ must match result, so units are $\frac{\text{mol}}{\text{cm}^3\text{s}}$

### Test Yourself
#### Problem 1
What is a dimensionally homogeneous equation? If an equation is dimensionally homogeneous, is it necessarily valid? If an equation is valid, is it dimensionally homogenous?
#### Solution 1
- Dimensionally homogeneous means dimensions match correctly (i.e. length/time + length/time - (length-length)/(time-length) )
- Equation must have same units in addition to correct dimensions; it is possible to be dimensionally homogeneous but still be invalid
- Yes, a valid equation must be dimensionally homogeneous

#### Problem 2
If $y(\text{m/s}^2) = az(\text{m}^3)$, what are the units of $a$?
#### Solution 2
$a$ must have units $1/(\text{m}^2 \text{s}^2)$

#### Problem 3
What is a dimensionless group? What multiplicative combination of $r(\text{m})$, $s(\text{m/s}^2)$, and $t(\text{s})$ would constitute a dimensionless grou?
#### Solution 3
- A dimensionless group has units/dimensions cancel to result in no dimensions for that term/group
- Many possibilities, one is $\frac{r}{t^2 s}$

#### Problem 4
If $z(\text{lb}_f) = a\text{sin}(Q)$, what are the units of $a$ and $Q$?
#### Solution 4
- Q must be dimensionless
- a must be $\text{lb}_f$